### ISSUE TITLE
When I run Tenseal tutorials 2 on raspberry pi 4 i always have error "module 'tenseal' has no attribute 'context'"

### ISSUE THREAD

**Issue by YanchoBasil-1989** [OpenMined/TenSEAL#299]
## Question
**Describe your question in ONE SENTENCE.**

## Further Information
Describe your question in greater length here.

## Screenshots
If applicable, add screenshots to help explain your question.

## System Information
 - OS Version: [e.g. 22]
 - Language Version: [e.g. Python 3.7, Node 10.18.1]
 - Package Manager Version: [e.g. Conda 4.6.1, NPM 6.14.1]

**Comment by youben11** [comment]
I don't think we have support to ARM processors, in which case, you are installing a really old package, which is just packaging some minimal source code (can you check the version of the tenseal package you have?). In order to have it working on a raspberry for the moment, you will need to build it from source. You can find instructions to do so in the README

**Comment by YanchoBasil-1989** [comment]
thanks very much will try that
